x-pack/plugins/security_solution/public/resolver/view/panels/cube_for_process.tsx

```diff
@@ -11,48 +11,71 @@ import { i18n } from '@kbn/i18n';
 /* eslint-disable react/display-name */
 
 import React, { memo } from 'react';
-import { useResolverTheme } from '../assets';
+import { useResolverTheme, SymbolIds } from '../assets';
+
+interface StyledSVGCube {
+  readonly isOrigin?: boolean;
+}
 
 /**
  * Icon representing a process node.
  */
 export const CubeForProcess = memo(function ({
   running,
+  isOrigin,
   'data-test-subj': dataTestSubj,
 }: {
   'data-test-subj'?: string;
   /**
    * True if the process represented by the node is still running.
    */
   running: boolean;
+  isOrigin?: boolean;
 }) {
   const { cubeAssetsForNode } = useResolverTheme();
-  const { cubeSymbol } = cubeAssetsForNode(!running, false);
+  const { cubeSymbol, strokeColor } = cubeAssetsForNode(!running, false);
 
   return (
-    <StyledSVG width="1.5em" height="1.5em" viewBox="0 0 1 1" data-test-subj={dataTestSubj}>
+    <StyledSVG
+      width="2.15em"
+      height="2.15em"
+      viewBox="0 0 100% 100%"
+      data-test-subj={dataTestSubj}
+      isOrigin={isOrigin}
+    >
       <desc>
         {i18n.translate('xpack.securitySolution.resolver.node_icon', {
           defaultMessage: '{running, select, true {Running Process} false {Terminated Process}}',
           values: { running },
         })}
       </desc>
+      {isOrigin && (
+        <use
+          xlinkHref={`#${SymbolIds.processCubeActiveBacking}`}
+          fill="transparent"
+          x={0}
+          y={-1}
+          stroke={strokeColor}
+          strokeDashoffset={0}
+          width="100%"
+          height="100%"
+          className="origin"
```

Comment:
Instead of using class names for this could you use this pattern instead: https://styled-components.com/docs/advanced#referring-to-other-components

This way we won't pick up unwanted styles from other code and we won't accidentally style code from other projects with our selectors.

Reply:
Yep, done. Thanks